src/domain/profile/children/child/ProfileChild.tsx

```diff
@@ -37,8 +37,9 @@ const ProfileChild: React.FunctionComponent<ProfileChildProps> = ({
 
   const upcomingEventsAndEventGroups =
     child.upcomingEventsAndEventGroups?.edges[0]?.node?.name;
-  const isNamed = Boolean(child.firstName);
-  const childName = `${child.firstName} ${child.lastName}`;
+  const name = child.name.split(' ');
+  const isNamed = Boolean(name[0]);
```

Comment:
wouldn't `!!name` do the same? Using index is a bit strange imho.

Reply:
fixed